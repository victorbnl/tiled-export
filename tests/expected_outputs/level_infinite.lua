return {
  version = "1.9",
  tiledversion = "1.9.2",
  class = "",
  orientation = "orthogonal",
  renderorder = "right-down",
  width = 10,
  height = 5,
  tilewidth = 48,
  tileheight = 48,
  backgroundcolor = {0, 170, 255},
  nextlayerid = 2,
  nextobjectid = 1,
  compressionlevel = -1,
  infinite = true,
  tilesets = {
    {
      name = "",
      firstgid = 1,
      filename = "tileset.tsx"
    }
  },
  layers = {
    {
      x = 0,
      y = 0,
      offsetx = 0,
      offsety = 0,
      parallaxx = 1,
      parallaxy = 1,
      opacity = 1,
      visible = true,
      id = 1,
      name = "Tile Layer 1",
      class = "",
      width = 10,
      height = 5,
      encoding = "lua",
      chunks = {
        {
          x = 0,
          y = 0,
          width = 16,
          height = 16,
          data = {
            0, 0, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0,
            1, 1, 0, 0, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0,
            3, 3, 0, 0, 3, 3, 3, 3, 3, 3, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
          }
        }
      }
    }
  },
  luaversion = "5.1"
}
