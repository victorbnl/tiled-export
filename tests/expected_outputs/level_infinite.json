{
  "version": "1.9",
  "tiledversion": "1.9.2",
  "class": "",
  "orientation": "orthogonal",
  "renderorder": "right-down",
  "width": 10,
  "height": 5,
  "tilewidth": 48,
  "tileheight": 48,
  "backgroundcolor": "#00aaff",
  "nextlayerid": 2,
  "nextobjectid": 1,
  "compressionlevel": -1,
  "infinite": true,
  "tilesets": [
    {
      "name": "",
      "firstgid": 1,
      "source": "tileset.tsx"
    }
  ],
  "layers": [
    {
      "x": 0,
      "y": 0,
      "offsetx": 0,
      "offsety": 0,
      "parallaxx": 1,
      "parallaxy": 1,
      "opacity": 1,
      "visible": true,
      "id": 1,
      "name": "Tile Layer 1",
      "class": "",
      "width": 10,
      "height": 5,
      "chunks": [
        {
          "x": 0,
          "y": 0,
          "width": 16,
          "height": 16,
          "data": [
            0, 0, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0,
            1, 1, 0, 0, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0,
            3, 3, 0, 0, 3, 3, 3, 3, 3, 3, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
          ]
        }
      ],
      "type": "tilelayer"
    }
  ],
  "type": "map"
}
